elyra/pipeline/component_parser_airflow.py

```diff
@@ -164,21 +152,15 @@ def get_runtime_specific_properties(self) -> List[ComponentParameter]:
         """
         Define properties that are common to the Airflow runtime.
         """
-        properties = [ComponentParameter(id="runtime_image",
-                                         name="Runtime Image",
-                                         type="string",
-                                         value="",
-                                         description="Container image used as execution environment.",
-                                         control="custom",
-                                         control_id="EnumControl",
-                                         required=True)]
-        return properties
-
-    def _read_component_definition(self, registry_entry: SimpleNamespace) -> str:
-        """
-        Delegate to ComponentReader to read component definition
-        """
-        reader = self._get_reader(registry_entry)
-        component_definition = reader.read_component_definition(registry_entry)
-
-        return component_definition
+        return [
+            ComponentParameter(
+                id="runtime_image",
+                name="Runtime Image",
+                type="string",
```

Comment:
I'm on a mission - thou shall not use the name `type` for anything! :smile:
```suggestion
                data_type="string",
```